test/batchGet.test.ts

```diff
@@ -0,0 +1,77 @@
+import TestContext, { documentClient } from './helpers/TestContext';
+import times from 'lodash.times';
+import { v4 as uuid } from 'uuid';
+
+let context: TestContext;
+const items: any[] = [];
+
+const testHashKey = uuid();
+
+beforeAll(async () => {
+  context = await TestContext.setup();
+
+  for (let i = 0; i < 10; i++) {
```

Comment:
You're already pulling in `lodash.times`, why not use that here.

Reply:
Thanks. Will fix this in a follow-up PR coming soon.